app/views/borrow.blade.php

```diff
@@ -181,9 +176,23 @@ function(data){
     }
   });
 
+  $("body").on("click", ".btn_delete", function() {
+    var id = $(this).prop('id');
+    var text = "{{ url('borrow/delete') }}/" + id;
+    $.ajax({
+      type: "GET",
+      url: text,
+    }).done(function(data) {
+      if(data['status']) {
+        $("#media-row_" + id).hide();
+      }
+    });
+  });
+
   $("body").on("click", ".book_choose", function(event){ 
       event.preventDefault();
       var text = $(this).prop('href');
+      var id = $(this).prop('id');
         console.log(text);
         $.ajax({
           type: "GET",

```

Comment:
I'll refactor all of 'GET' on ajax to 'POST' on further issues. ( refactor period. )